{{ cookiecutter.repo_name }}/.github/workflows/ci.yml

```diff
@@ -3,21 +3,23 @@ name: Continuous Integration
 on: [push]
 
 jobs:
-    test:
-      runs-on: ubuntu-latest
-      container:
-        image: docker://{{ cookiecutter.base_docker_image }}
-      steps:
+  test:
+    runs-on: ubuntu-latest
+    steps:
       - name: Checkout
         uses: actions/checkout@v1
       - name: Black
-        run: black --check {{ cookiecutter.repo_name }}
-      - name: Flake8 
-        run: flake8 {{ cookiecutter.repo_name }}
+        uses: ./docker/action
```

Comment:
@jeffmm Is this a new thing? I don't think GitHub actions has this before.

Reply:
I'm not sure how new any of the Docker integration features are, but I assume private actions have always been a thing.